invenio_mail/templates/invenio_mail/base.html

```diff
@@ -0,0 +1,40 @@
+{#
+# This file is part of Invenio.
+# Copyright (C) 2016 TIND.
```

Comment:
Please check if you are willing to donate your contribution to CERN, so that CERN remains the copyright holder. Otherwise the copyright statements in other files also needs to be fixed.